### DISCUSSION TITLE
Ability to create project template, and clone data

### DISCUSSION THREAD

**Discussion by wallentx** [community/community#14575] [Projects and Issues]
I set up a project template for dogfooding our builds. I'm getting away with using Draft Issues as individual cell descriptions, such as "Test Installer", so I hope I don't lose that ability.
I made a view for each OS we need to test our builds against, which means I had to make a draft issue for "Test Installer" for each OS. It would be nice to "clone view" in addition to "duplicate view", where "clone view" is a fork of the view data, rather than a linked copy.

I think for our purposes, this would work fine if we could have a saved "template" project, and have the ability to clone that template, so that we could create new projects from template, and dispose of the old one for each build we need to dogfood.

**Comment by lucalves** [comment]
Hi @wallen, how are you? 

I don't know if I understood correctly, but see if this solution of creating a template repository will help you: https://docs.github.com/pt/repositories/creating-and-managing-repositories/creating-a-template-repository

**Reply by wallentx** [reply]
Hi, sorry, I was referring specifically to the Projects (beta) https://docs.github.com/en/issues/trying-out-the-new-projects-experience

**Comment by archon810** [comment]
Related: https://github.com/github-community/community/discussions/7908.

**Comment by rileybroughten** [comment]
👋 Wanted to drop in here to say that project templates are now in public beta and we would love your feedback! Here is a link to the [documentation](https://docs.github.com/en/issues/planning-and-tracking-with-projects/managing-your-project/managing-project-templates-in-your-organization), as well as the [discussion link](https://github.com/orgs/community/discussions/54576) for any feedback that you may have. Please let us know how it works for you and where else we can improve! 🎉

**Reply by rileybroughten** [reply]
[Changelog](https://github.blog/changelog/2023-05-25-github-issues-projects-may-25th-update/) announcement here as well 👍
